srcpkgs/exa/template

```diff
@@ -3,28 +3,33 @@ pkgname=exa
 version=0.10.1
 revision=2
 build_style=cargo
-hostmakedepends="pkg-config"
+hostmakedepends="pkg-config $(vopt_if man pandoc)"
 makedepends="libgit2-devel"
 short_desc="Modern replacement for ls"
 maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
 license="MIT"
 homepage="https://the.exa.website/"
-distfiles="https://github.com/ogham/exa/archive/refs/tags/v${version}.tar.gz
- https://github.com/ogham/exa/releases/download/v${version}/exa-accoutrements-v${version}.zip"
-checksum="ff0fa0bfc4edef8bdbbb3cabe6fdbd5481a71abbbcc2159f402dea515353ae7c
- 531596a1ef2a757c7728087529528150e6eb52bb8224fe575aa00a5f1b762849"
+distfiles="https://github.com/ogham/exa/archive/refs/tags/v${version}.tar.gz"
+checksum="ff0fa0bfc4edef8bdbbb3cabe6fdbd5481a71abbbcc2159f402dea515353ae7c"
 
-post_extract() {
-	mkdir -p accoutrements
-	mv ../man accoutrements/
-	mv ../completions accoutrements/
-}
+build_options="man"
+desc_option_man="Use pandoc for manpages"
+
+case "$XBPS_MACHINE" in
+	x86_64*|i686|ppc64le*|ppc64) build_options_default="man" ;;
```

Comment:
because of this restriction, aren't pre-generated manpages better?

Reply:
I can take it back, but is that such a big deal?

Reply:
All cross compiled architectures don't have pandoc actually, this condition isn't exactly correct.

Reply:
Reverted to previous solution, sorry for the commotion